inst/include/dust/dust.hpp

```diff
@@ -83,14 +79,17 @@ class Dust {
     for (auto i : shape) {
       shape_.push_back(i);
     }
+#ifdef DUST_USING_CUDA_PROFILER
+    cuda_profiler_start(device_config_);
+#endif
   }
 
   // We only need a destructor when running with cuda profiling; don't
   // include ond otherwise because we don't actually follow the rule
   // of 3/5/0
-#ifdef DUST_ENABLE_CUDA_PROFILER
+#ifdef DUST_USING_CUDA_PROFILER
```

Comment:
Also unnecessary

Reply:
This one is necessary, as we don't generally want the destructor - I've put the other ones in for symmetry, but it still feels pretty nasty. I did wonder about putting the profile control into the control object but had issues with the destructor there too...